store/models/models.go

```diff
@@ -97,3 +97,24 @@ type Comment struct {
 func newComment(r *ReviewEvent, c *lookout.Comment) *Comment {
 	return &Comment{ID: kallax.NewULID(), ReviewEvent: r, Comment: *c}
 }
+
+// Organization is a persisted model for an Organization (e.g. a GitHub App
+// installation). It contains settings for a group of repositories.
+// The primary key should be (Provider,InternalID), but kallax does not support
+// composite or string keys
+type Organization struct {
```

Comment:
If `go-kallax` does not support unique keys for more than 1 column, I'd also say so in the docs

Reply:
ok, done in cf97909